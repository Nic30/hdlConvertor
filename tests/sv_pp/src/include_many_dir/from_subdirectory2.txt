`include "dir0-a/dir1-a/file2-b.txt"
`include "./dir0-a/dir1-a/file2-b.txt"
`include "./dir0-a/../dir0-a/dir1-a/file2-b.txt"
