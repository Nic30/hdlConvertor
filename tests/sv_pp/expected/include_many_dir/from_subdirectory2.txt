"file2-a.txt"
"file2-a.txt"
"file2-a.txt"
